src/data/redirects.ts

```diff
@@ -399,6 +399,11 @@ const redirectsData = [
         destination: "/admin/observability/alerting#set-up-alerts-in-grafana",
         permanent: true
     },
+    {
+        source: "/@v5.3.0/admin/observability/alerts",
+        destination: "https://docs.sourcegraph.com/@v5.3.0/admin/observability/alerts#worker-perms-syncer-sync-errors",
+        permanent: false
```

Comment:
Set this to `false` to avoid shooting ourselves in the foot on this one, in case we did a mistake. I'd be cautious to set that one to true before we actually handle the versioning.